### ISSUE TITLE
Bring Gradle to java-10

### ISSUE THREAD

**Issue by jeandersonbc** [javapathfinder/jpf-core#138]
As Gradle is now the standard build tool for JPF, the future release (Java 10) must have Gradle support as well.

**Comment by jeandersonbc** [comment]
Curretly, the gradle branch is in-sync with the java-10 branch.
In [this link](https://www.diffchecker.com/ZtwhNVZU), there's a diff with the Travis log for java-10 (left) and for gradle (right).

Next step should be effectively updating gradle script to build and test JPF

**Comment by cyrille-artho** [comment]
Is it possible to configure the diff such that some lines are ignored?
Normal diff has the `-I` option for this.

**Comment by jeandersonbc** [comment]
I used this online tool because it was better to visualize the difference.

I can remove the elapsed time to avoid noise and try again with the command
line tool and the "-I" option.

On Aug 2, 2018 11:48 AM, "cyrille-artho" <notifications@github.com> wrote:

Is it possible to configure the diff such that some lines are ignored?
Normal diff has the -I option for this.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub
<https://github.com/javapathfinder/jpf-core/issues/138#issuecomment-409952762>,
or mute the thread

**Comment by gayanW** [comment]
Hi @jeandersonbc. It is good to hear that the two branches are in sync.

Here is a diff which shows all the changes that we've made to build.xml since the initial commit. It might be of help.

https://pastebin.com/fNCD6evF

**Comment by jeandersonbc** [comment]
I uploaded both logs without the `Time Elapsed: ...` field to remove the noise from the different measurements.

@cyrille-artho I ran `sdiff` and, in summary, those are the differences:
- gradle branch has our `ReporterResourcesTest`
- MethodTest was updated (one contributor added a test that was merged)

For reference: https://gist.github.com/jeandersonbc/a38250fc22b6228018a9a25b8603dff1#file-sdiff-output

I believe I'm good to go. I will keep working on the gradle branch but keeping it in sync with java-10 for future merge.
